src/app/components/message/content/ImageContent.tsx

```diff
@@ -172,6 +177,35 @@ export const ImageContent = as<'div', ImageContentProps>(
             })}
           </Box>
         )}
+        {srcState.status === AsyncStatus.Success && spoiled && (
+          <Box className={css.AbsoluteContainer} alignItems="Center" justifyContent="Center">
+            <TooltipProvider
+              tooltip={
+                typeof spoilerReason === 'string' && (
+                  <Tooltip variant="Secondary">
+                    <Text>{spoilerReason}</Text>
+                  </Tooltip>
+                )
+              }
+              position="Top"
+              align="Center"
+            >
+              {(triggerRef) => (
+                <Button
+                  ref={triggerRef}
+                  variant="Secondary"
+                  fill="Solid"
+                  radii="300"
+                  size="300"
```

Comment:
I think we should instead use a `Chip` of `500` size here (because media autoload disable shows a button) and probably giving it a `Warning` variant with `Pill` radii.

Reply:
I've kept the variant as-is because the amber color of the `Warning` variant doesn't stand out against similarly-colored spoiled images.